packages/flutter_tools/flutter_analysis_options

```diff
@@ -10,6 +10,8 @@
 # need to be silenced, in particular, public_member_api_docs.
 #
 # When editing, make sure you keep /.analysis_options consistent.
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
```

Comment:
I'd put this at the end of the first paragraph, rather than the end of the whole block. In particular, I'd recommend leaving the "When editing" line at the end of the block for visibility.